src/pages/myAdmin/guides/gettingStarted.tsx

```diff
@@ -19,22 +18,21 @@ const accessApi: ReactNode = (
         <p>
             The API supports POST requests only.
         </p>
-        <h2>Test Environment</h2>
+        <h2>Test environment</h2>
         <p>
             A sandbox environment is a testing environment that is available to developers who wish to test their application before making API calls to the live system. The test environment can be accessed at:
         </p>
         <code className="small-code-sample">https://myadminapitest.geotab.com/v2/MyAdminApi.ashx</code>
         <p>
             A few methods are not supported in the test environment and will throw exceptions when called. For more information, see the{" "}
-            {/* TODO: update link */}
-            <Link to="">Reference</Link>{" "}
+            <Link to="./../../apiReference/methods">Reference</Link>{" "}
```

Comment:
also this one

Reply:
Has been updated to "/myAdmin/apiReference/methods"